dubbo-plugin/dubbo-qos/src/main/java/org/apache/dubbo/qos/server/Server.java

```diff
@@ -115,17 +115,33 @@ protected void initChannel(Channel ch) throws Exception {
                 ));
             }
         });
-        try {
-            if (StringUtils.isBlank(host)) {
-                serverBootstrap.bind(port).sync();
+        Throwable lastError = null;
+        for (int i = 0; i < 10; i++) {
```

Comment:
I recommend adding a server-failed-retry-count  and  server-failed-retry-interval configuration

Reply:
ok

Reply:
done